Tutorials/Creating-Basic-Site/Articles-Parent-and-Article-Items/index.md

```diff
@@ -2,109 +2,132 @@
+    ![Query parameters](images/query-parameters.png)
+8. If you've set the correct parameters, you will get a preview of the items being selected with the query. Click **Submit**, and you will see a code snippet has been added to your template.
+    It will look similar to this:
+    ```csharp
+    @{
+        var selection = Umbraco.Content(Guid.Parse("c5ed4ea3-f9ea-4c37-a798-fe7892c98b95"))
+        .ChildrenOfType("articlesItem")
+        .Where(x => x.IsVisible())
+        .OrderByDescending(x => x.CreateDate);
     }
-</ul>
-```
-
-This code will output a list of all the **_Article Items_** as links using the name. We are going to modify this a little, to add a bit more information about the articles. Replace the `HTML` in the *foreach* loop with this snippet:
-
-```csharp
-<article class="special">
-        <div class="articletitle"><a href="@item.Url">@item.Name</a></div>
-        <div class="articlepreview">@Html.Truncate(item.Value("articleContent").ToString(), 20, true)<a href="@item.Url">Read More..</a></div>
-    </article>
-<hr/>
-```
-
-Now check this in the browser!
-
+    <ul>
+        @foreach (var item in selection)
+        {
+            <li>
+                <a href="@item.Url">@item.Name</a>
+            </li>
+        }
+    </ul>
+    ```
+9. The above code will output a list of all the **_Article Items_** as links using the name. We will modify the template a little, to add more information about the articles. Replace the `HTML` in the *foreach* loop with this snippet:
+
+    ```csharp
+    <article class="special">
+            <div class="articletitle"><a href="@item.Url">@item.Name</a></div>
+            <div class="articlepreview">@Html.Truncate(item.Value("articleContent").ToString(), 20, true)<a href="@item.Url">Read More..</a></div>
+        </article>
+    <hr/>
+    ```
+10. Click **Save**.
+
+To update the **Articles Item** template, follow these steps:
+
+1. Go to **Settings**.
+2. In the **Templating** section, expand the **Templates** folder. You should see a template titled **_Articles Item_**.
+3. In the **Master template** select **Master** and click **Save**.
+4. Copy the template content from the **_Simple Content Page_**  template and paste the content into **Articles Item** template.
+    
+    :::warn
+    Take care when pasting the template not to overwrite the first line `@inherits Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.ArticlesMain>`. If you get an error when loading the page ensure the last part in <> brackets matches your Document Type alias.
+    :::
+5. Replace the static text within the `<p>` tags with the Model.Value references. For example:  Add references to the **_articleTitle_** and **_articleContent_**.
+    ![Articles Item Template](images/articles-item-template.png)
+6. Click **Save**.
+
+Check your browser, you should now see something similar to the screen below.
 ![Finished Articles section](images/article-main-frontend.png)
```

Comment:
Looks like something in the code snippet to preview the articles on the "ARticle Main" page, is missing... 
When I use the snippets, I do not get the "Article Title" shown, like it is in the image here 🤔